zebra-chain/src/transaction/unmined.rs

```diff
@@ -230,13 +244,17 @@ pub struct UnminedTx {
     pub conventional_fee: Amount<NonNegative>,
 }
 
+impl fmt::Debug for UnminedTx {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // Logging unmined transactions can leak sensitive user information,
+        // particularly when Zebra is being used as a `lightwalletd` backend.
+        f.debug_tuple("UnminedTx").field(&"private").finish()
```

Comment:
Should this call `self.to_string()` like `Unmined` does above? Seems like a nice pattern

Reply:
I'm not too concerned about the difference here, happy if you want to make another PR to fix it up.